OverlayPlugin.Core/ClipboardTechSupport.cs

```diff
@@ -106,7 +108,8 @@ public ClipboardTechSupport(TinyIoCContainer container)
                 settings.Add(new List<string> { "Machina Region", repository.GetMachinaRegion().ToString() });
                 string gameVersion = repository.GetGameVersion();
                 settings.Add(new List<string> { "Game Version", gameVersion != "" ? gameVersion : "(not running)" });
-                settings.Add(new List<string> { "Screen Mode", GetFFXIVScreenMode(repository.GetCurrentFFXIVProcess()) });
+                repository.RegisterProcessChangedHandler(GetFFXIVScreenMode);
+                settings.Add(new List<string> { "Screen Mode", screenMode });
```

Comment:
I'm pretty sure this will leak memory, every time the tech support button is clicked a new process change handler will be registered but never cleared so it'll stay in memory?

This may need to be a static handler, something like:
```cs
private static string screenMode = null;
// ...
if (screenMode == null) {
    repository.RegisterProcessChangedHandler(GetFFXIVScreenMode);
}
settings.Add(new List<string> { "Screen Mode", screenMode });
```
}

Reply:
Oh, fair, I wasn't thinking about multiple clicks.